### PR TITLE
{bio}[GCC/11.2.0] AGAT 0.9.2

### PR DESCRIPTION
**Pull request by fizwit**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/a/AGAT/AGAT-0.9.2-GCC-11.2.0.eb [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+
+builddependencies = [('binutils', '2.37')]
+
+dependencies = [
+    ('Perl', '5.34.0'),
+    ('BioPerl', '1.7.8'),
+]
+
+exts_defaultclass = 'PerlModule'
+exts_filter = ("perldoc -lm %(ext_name)s ", "")
```
Comment (smoors) [inline easybuild/easyconfigs/a/AGAT/AGAT-0.9.2-GCC-11.2.0.eb:24]:
```suggestion
exts_filter = ("perl -e 'require %(ext_name)s'", '')
```

File: easybuild/easyconfigs/a/AGAT/AGAT-0.9.2-GCC-11.2.0.eb [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+        'checksums': ['998e32a3cb5e941efd324daf9ee4e0a71d605d06999922f083170b7a9f059899'],
+    }),
+]
+
+modextrapaths = {'PERL5LIB': 'lib/perl5/site_perl/%(perlver)s/'}
+
+sanity_check_paths = {
+    'files': [],
+    'dirs': ['bin', 'lib/perl5/site_perl/%(perlver)s/%(name)s'],
+}
```
Comment (smoors) [inline easybuild/easyconfigs/a/AGAT/AGAT-0.9.2-GCC-11.2.0.eb:65]: can you add `sanity_check_commands` here for some of the AGAT tools? e.g.
```
sanity_check_commands = ['agat_xxx --help']
```

Comment (fizwit) [reply easybuild/easyconfigs/a/AGAT/AGAT-0.9.2-GCC-11.2.0.eb:65]: sounds good,

File: easybuild/easyconfigs/a/AGAT/AGAT-0.9.2-GCC-11.2.0.eb (hunk 1 above)
Comment (smoors) [reply easybuild/easyconfigs/a/AGAT/AGAT-0.9.2-GCC-11.2.0.eb]: @fizwit thanks for adding the sanity_check_commands.
I made the above change for you, as this is a more robust way to check if a perl module is correctly installed.

**Comment by smoors** [comment]
@boegelbot: please test @ generoso

**Comment by smoors** [comment]
Test report by @smoors
**SUCCESS**
Build succeeded for 3 out of 3 (1 easyconfigs in total)
node301.hydra.os - Linux CentOS Linux 7.9.2009, x86_64, Intel(R) Xeon(R) Gold 6148 CPU @ 2.40GHz (skylake_avx512), Python 2.7.5
See https://gist.github.com/47cf672b12667076afc86927b0e6ffb8 for a full test report.

**Review by smoors — APPROVED** [review]
lgtm

**Comment by smoors** [comment]
Going in, thanks @fizwit!

**Pull request merged**